src/DrugstoreApp/Backend/DrugstoreAPI/DrugstoreAPI/Adapters/MedicineAdapter.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DrugstoreAPI.Dtos;
+using DrugstoreAPI.Models;
+
```

Comment:
Try with AutoMapper

Reply:
Ne mozemo da stignemo to sad, sredicemo u sledecem sprintu.